pkg/segment/structs/segstructs.go

```diff
@@ -145,6 +146,8 @@ type TransactionGroupState struct {
 	Timestamp uint64
 }
 
+// Update this function: GetAllColsInAggsIfStatsPresent() to return all columns in the query aggregators if stats are present.
+// This function should return all columns in the query aggregators if stats are present.
 type QueryAggregators struct {
```

Comment:
I'm a little confused by this comment. Are you saying if we add fields to `QueryAggregators` we might need to update `GetAllColsInAggsIfStatsPresent()`?